system_test/common/aest_db_SUITE.erl

```diff
@@ -93,11 +103,42 @@ node_db_host_path(NodeName, Config) ->
 format(Fmt, Args) ->
     iolist_to_binary(io_lib:format(Fmt, Args)).
 
+run_db_rename_fun(DbHostPath, Cfg) ->
+    case proplists:lookup(rename_db_fun, Cfg) of
+        none ->
+            ok;
+        {rename_db_fun, Fun} ->
+            Fun(DbHostPath, Cfg)
+    end.
+
+run_rename_db_script(DbHostPath, Cfg) ->
+    N3 = node_spec_custom_entrypoint(node3, DbHostPath),
+    aest_nodes:setup_nodes([N3], Cfg),
+    aest_nodes:start_node(node3, Cfg),
+    aest_nodes:run_cmd_in_node_dir(node3, ["bin/aeternity", "console"], #{timeout => 5000}, Cfg),
```

Comment:
I need to understand this.

Reply:
Rather than this line, I would expect hardcoded expectation that the schema file is present already at the expected location (used in next line).

Reply:
So this line is potentially creating persisted DB with new name in it.

Reply:
I see this is now removed. 👍